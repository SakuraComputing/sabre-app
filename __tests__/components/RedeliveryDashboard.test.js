import React from 'react';
import { shallow } from 'enzyme';
import RedeliveryHistory from '../../src/components/RedeliveryDashboard';

describe('Redelivery History',() => {
    it('should render the form correctly',() => {
        const wrapper = shallow(<RedeliveryHistory/>);
        expect(wrapper).toMatchSnapshot();
    })
});


