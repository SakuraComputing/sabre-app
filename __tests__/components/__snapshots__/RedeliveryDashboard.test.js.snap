// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Redelivery History should render the form correctly 1`] = `
<header>
  <h2>
    Redeliveries
  </h2>
  <p />
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/create"
  >
    Add Redelivery
  </NavLink>
  <div>
    <Connect(RedeliveryHeader) />
    <Connect(RedeliveryList) />
  </div>
</header>
`;
